apiserver/facades/client/spaces/register.go

```diff
@@ -37,11 +38,15 @@ func newAPI(ctx facade.ModelContext) (*API, error) {
 	credentialInvalidatorGetter := credentialcommon.CredentialInvalidatorGetter(ctx)
 	check := common.NewBlockChecker(st)
 	auth := ctx.Auth()
+
+	providerGetter := facade.ProviderRunner[environs.NetworkingEnviron](ctx)
+
 	return newAPIWithBacking(apiConfig{
 		NetworkService:              ctx.ServiceFactory().Network(),
 		Backing:                     stateShim,
 		Check:                       check,
 		CredentialInvalidatorGetter: credentialInvalidatorGetter,
+		ProviderGetter:              providerGetter,
```

Comment:
Can this field also be removed from the struct definition?

Reply:
No? This is what's being used to replace the EnvironConfigGetter